Specs/TestWorkers/throwError.js

```diff
@@ -1,12 +1,10 @@
 define([
-        'Core/RuntimeError',
         'Workers/createTaskProcessorWorker'
     ], function(
-        RuntimeError,
         createTaskProcessorWorker) {
     'use strict';
 
     return createTaskProcessorWorker(function(parameters, transferableObjects) {
-        throw new RuntimeError(parameters.message);
+        throw new Error(parameters.message);
```

Comment:
Why no longer using `RuntimeError` here?

Reply:
Because these test-only workers don't go through any kind of build process and `Core/RuntimeError` won't be available to them as part of the built Cesium.js tests.  There was no reason for them to use RuntimeError so it was easiest to just remove it.